ui/components/app/confirm-page-container/confirm-detail-row/index.scss

```diff
@@ -58,4 +58,12 @@
       width: 90px;
     }
   }
+
+  .ledger-live-dialog {
```

Comment:
ToDo: Upon design confirmation use `<Typography>` components

Reply:
Nit, but should this be `.confirm-detail-row__ledger-live-dialog` and `.confirm-detail-row__ledger-live-dialog-step` instead, since they're child elements of `.confirm-detail-row` and not components on their own?

Reply:
@mcmire - I've updated the PR to use our design system typography components, so this css has been removed, thanks!